src/seedu/addressbook/Main.java

```diff
@@ -55,7 +55,7 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | StorageFileNotFoundException e) {
```

Comment:
Incorrect attempt on handling the exception. 

The response (the error message) may not be appropriate, the showInitFailedMessage() inform the users that there is something wrong while the program is starting up (initialization), but this file missing exception can happen throughout the execution period. So, you may want to handle this separately.